lib/insights_cloud/async/connector_playbook_execution_reporter_task.rb

```diff
@@ -154,7 +156,11 @@ def report_job_progress(invocation_status)
             all_hosts_success &&= status['exit_status'] == 0
           end
         end
-        generator.job_finished_message(all_hosts_success) if done?(invocation_status)
+
+        if (job_invocation.finished? || hosts.all { |h| h.done? })
```

Comment:
I suppose you meant all hosts reported "done" message:
```suggestion
        if (job_invocation.finished? || all_hosts_success)
```